src/swa.d.ts

```diff
@@ -68,7 +68,7 @@ declare type SWACLIConfig = GithubActionWorkflow & {
   swaConfigFilename?: "staticwebapp.config.json";
   swaConfigFilenameLegacy?: "routes.json";
   app?: string;
-  api?: string;
+  apiLocation?: string;
```

Comment:
any reason for this renaming?

Reply:
https://github.com/Azure/static-web-apps-cli/blob/main/src/cli/index.ts#L34

```ts
.option("--api, --api-location <apiLocation>", "set the API folder or Azure Functions emulator address", DEFAULT_CONFIG.apiLocation)
```

In the option we specify the option name as `apiLocation`. And this is what the property is named in `cli.opts()`. So in order to easily override this value, I've matched the property names.

We could add logic to allow us to use either `api` or `apiLocation` in the config file, but that might confuse users even more.

Reply:
That makes sense. If you think this is gonna be confusing for users, then we can remove the shorthand flags `app` and `api` and just leave the flags that matched the ones SWA uses.

WDYT?

Reply:
I think that would be more clear. Do you want to make that change in this PR or a new PR?

Reply:
This would be a breaking change for any commands already setup to use `--api` and `--app`, are we ok with that?

Reply:
I think this could be done in a different PR. 

FWIW, the CLI is still in _preview_ stage. But we can:
- plan any breaking change in version X+1.
- Notify the users (in version X) about this upcoming breaking change, if we detect any deprecated flag/command.

cc @Reshmi-Sriram

Reply:
Sounds good. I will leave this PR as-is and plan on making these changes in another PR after this one is merged.